README.md

```diff
@@ -142,47 +149,42 @@ When you open a new pull request, a bot will evaluate whether you have signed th
 
 If the `license/cla` status check remains on *Pending*, even though all contributors have accepted the CLA, you can recheck the CLA status by visiting the following link (replace **[PRID]** with the ID of your PR): <https://cla-assistant.io/check/demisto/demisto-py?pullRequest=[PRID>] .
 
+---
+
 ## Dev Environment Setup
 
-We recommend installing both development environments. We use [tox](https://github.com/tox-dev/tox) for managing environments and running unit tests.
+We will now setup a quick virtualenv in which we will install the `demisto-py` version you are currently working on.
+This will be used as your testing environment, you do not need to update it again or re-run in any way.
 
-Install `tox`:
+1. Make sure you have python3 installed.
 
-```bash
-pip install tox
-```
+2. Make sure you have [Poetry](https://python-poetry.org/) installed.
 
-List configured environments:
+3. run `poetry install`
 
-```bash
-tox -l
-```
+4. For further reading about Poetry, you can refer to [Poetry documentation](https://python-poetry.org/).
 
-Then setup dev virtual envs for python 3 (will also install all necessary requirements):
+You have now setup the your `demisto-py` dev environment!
 
-```bash
-tox --devenv py310
-```
+To activate it simply run: `poetry shell`.
 
-Switch to python 3 env by running:
+To deactivate the virtual environment and return simply run: `exit`.
```

Comment:
not `deactivate`?

Reply:
`To deactivate the virtual environment and exit this new shell type exit.` 
`To deactivate the virtual environment without leaving the shell use deactivate.`

[poetry-docs](https://python-poetry.org/docs/basic-usage/#activating-the-virtual-environment)